kernel/yosys.cc

```diff
@@ -149,16 +149,18 @@ void yosys_banner()
 	log("\n");
 }
 
-int ceil_log2(int x)
+int ceil_log2(const int x)
```

Comment:
I didn't look into the rest of the PR before: What are you hoping to achieve with this `const`?

Reply:
Just adds clarity to the code. I can remove it if you'd like.

Reply:
Not sure what it clarifies tho.. Please remove it.

Reply:
Done